fe/fe-core/src/main/java/org/apache/doris/catalog/TabletInvertedIndex.java

```diff
@@ -641,6 +644,9 @@ public Replica getReplica(long tabletId, long backendId) {
         try {
             Preconditions.checkState(tabletMetaMap.containsKey(tabletId),
                     "tablet " + tabletId + " not exists, backend " + backendId);
+            if (Config.isCloudMode()) {
```

Comment:
ditto

Reply:
because addReplica had use backend id = -1.

for cloud mode, tablet invert index just no need backend.